### DISCUSSION TITLE
Mobília Animal

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#323] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/mobilia-animal/

**Comment by brunoaugusto** [comment]
Mesmo não tendo gostado da série, me lembrou do Lord Tubbington de Glee



![null](<https://i.postimg.cc/Xvjz95Db/Lord-Tubbington.png>)

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/mobilia-animal/#comment_1128956062>):

> Mesmo não tendo gostado da série, me lembrou do Lord Tubbington de Glee[![null](<https://camo.githubusercontent.com/6ed239c47f5e73fa294a6a8ee1d4468469435de723a3d3c4fec377ef07b9c2f4/68747470733a2f2f692e706f7374696d672e63632f58766a7a393544622f4c6f72642d54756262696e67746f6e2e706e67>)](<https://camo.githubusercontent.com/6ed239c47f5e73fa294a6a8ee1d4468469435de723a3d3c4fec377ef07b9c2f4/68747470733a2f2f692e706f7374696d672e63632f58766a7a393544622f4c6f72642d54756262696e67746f6e2e706e67>)

Gato gordo, fumante, viciado drogas e apostas, membro de gangue, ex-presidiário e escravo doméstico que de alguma forma sabe ler e usar um computador :rofl:

**Comment by one-eyed-willy** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/mobilia-animal/#comment_1129000731>):

> Gato gordo, fumante, viciado drogas e apostas, membro de gangue, ex-presidiário e escravo doméstico que de alguma forma sabe ler e usar um computador 🤣

Ô loco! :open\_mouth:

**Comment by tali-zorah** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/mobilia-animal/#comment_1129000731>):

> Gato gordo, fumante, viciado drogas e apostas, membro de gangue, ex-presidiário e escravo doméstico que de alguma forma sabe ler e usar um computador 🤣

*Keelah!*
